graphql/schema.graphql

```diff
@@ -259,6 +280,13 @@ input UserInputType {
   last_signed_in: Float!
 }
 
+input UserPersonalizationInputType {
+  id: String!
+  user_id: String!
```

Comment:
We shouldn't need to pass `user_id` in from the client explicitly. Instead, we should use the ID of the currently authenticated user. This way, a given user can only mutate that user's personalizations.

Reply:
Good point. 
UserPersonalizationInputType can removed entirely and {id: id, user_id:user.uid, type:'frequency', value:'daily'} can be passed to upsertUserPersonalization mutation.
I will push this change in my next commit.